tests/containers/docker_image.pm

```diff
@@ -23,21 +25,21 @@ use utils;
 use containers::common;
 use containers::container_images;
 use suse_container_urls 'get_suse_container_urls';
-use scheduler 'get_test_suite_data';
+use version_utils qw(get_os_release check_host_os);
 
 sub run {
     my ($image_names, $stable_names) = get_suse_container_urls();
-    my $test_data = get_test_suite_data();
-    my $runtime   = "docker";
+    my ($running_version, $sp, $host_distri) = get_os_release;
+    my $runtime = "docker";
 
-    install_docker_when_needed($test_data->{host_os});
+    install_docker_when_needed($host_distri);
     allow_selected_insecure_registries(runtime => $runtime);
     scc_apply_docker_image_credentials() if (get_var('SCC_DOCKER_IMAGE'));
 
     for my $iname (@{$image_names}) {
         test_container_image(image => $iname, runtime => $runtime);
         build_container_image(image => $iname, runtime => $runtime);
-        unless ($test_data->{host_os}) {
+        if (check_host_os) {
```

Comment:
```suggestion
        if (check_host_os()) {
```
I know that perl allow your version too , but it take some time to figure out if it is a function or variable :(

Reply:
I'd even suggest `check_host_os('suse')`.

The `check_host_os()` should imho not have default value for its `$distri_name` and should `die` when no value is provided.

Reply:
Changed here as well as to podman_images

Reply:
I still don't understand what `check_host_os()` does. The argument is missing.
It makes no sense to have `'suse'` as default value of `$distri_name` and the `check_host_os()` should die if no argument is provided.